measured.gemspec

```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |spec|
   spec.authors       = ["Kevin McPhillips"]
   spec.email         = ["github@kevinmcphillips.ca"]
```

Comment:
Not really related to this patch, but shall we also add `gems@shopify.com` like [other Shopify gems](https://github.com/Shopify/identity_cache/blob/master/identity_cache.gemspec#L6)?

Reply:
Yes. I can change it. It's all out of date.

Reply:
Perfect :)